.github/workflows/ci.yml

```diff
@@ -43,10 +43,39 @@ jobs:
           ruby-version: "3.0"
           bundler-cache: true # runs 'bundle install' and caches installed gems automatically
 
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        include:
+          - platform: aarch64-linux
+        with:
+          install: true
+
+      - name: Enable Multiarch
+        run: |
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+        include:
```

Comment:
I think you want to use an "if" tag here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif